tests/test_SequenceManager.cc

```diff
@@ -529,3 +533,266 @@ TEST_CASE("SequenceManager: store_sequence() & load_sequence() - Empty sequence"
         REQUIRE(seq_deserialized.empty());
     }
 }
+
+
+TEST_CASE("SequenceManager: git repository", "[SequenceManager]")
+{
+
+    SECTION("Create and store sequence")
+    {
+        // clean surrounding
+        std::filesystem::remove_all(temp_dir);
+
+        // init manager
+        SequenceManager manager{ temp_dir};
+
+        // create sequence
+        Sequence seq{ "git Sequence 1",  SequenceName{ "git_sequence_1" }, UniqueId{ 0xabcdef123456 } };
+        Step step_01{Step::type_while};
+        step_01.set_label("while");
+        step_01.set_script("return i < 10");
+        seq.push_back(step_01);
+
+        // store sequence and make git commit
+        manager.store_sequence(seq);
+
+        // unable to reach gl debug functions without a new function in sequenceManager
+        // Therefore a standalone git handler has to be initialized
+        git::GitRepository gl{temp_dir};
+
+        // check if commit was done
+        const std::string& expected_msg = "change sequence:\n- Add sequence.lua from new sequence 'git_sequence_1[0000abcdef123456]'\n- Add step_1_while.lua from new sequence 'git_sequence_1[0000abcdef123456]'";
+        const auto& last_msg = gl.get_last_commit_message();
```

Comment:
Why do we create string *references* in these two lines instead of simple strings? This is just asking for trouble...